docs/applications/simple-bank-account/README.md

```diff
@@ -207,16 +207,15 @@ try (ClientService clientService = injector.getInstance(ClientService.class)) {
 
 ```java
 JsonObject argument = Json.createObjectBuilder().add("id", "010-123456789").build();
-ContractExecutionResponse response = clientService.executeContract("create-account", argument);
+ContractExecutionResult response = clientService.executeContract("create-account", argument);
```

Comment:
```suggestion
ContractExecutionResult result = clientService.executeContract("create-account", argument);
```

Reply:
Thank you for review comments.
Changes updated in 07087f3